onboarding_v2/welcome.md

```diff
@@ -9,7 +9,7 @@
 </ul>
 <p>&nbsp;</p>
 <p>Have questions?&nbsp; &nbsp;
-<a href="https://docs.wavefront.com/tutorial_overview.html"><img src="images/tut_snag.png"></img></a> &nbsp; &nbsp;<a href="https://help.wavefront.com/hc/en-us/requests/new"> <img src="images/get_help_snag.png"></img></a></p>
+<a href="https://docs.wavefront.com/tutorial_overview.html" aria-label="Tutorial and videos"><img src="images/tut_snag.png" alt="Tutorial icon"></img></a> &nbsp; &nbsp;<a href="https://help.wavefront.com/hc/en-us/requests/new" aria-label="Get help"> <img src="images/get_help_snag.png" alt="Help icon"></img></a></p>
```

Comment:
`aria-label="Tutorial and videos"` -> `aria-label="Tutorials and videos"`
`alt="Tutorial icon"` -> `alt="Video icon"`